sycl/plugins/unified_runtime/ur/adapters/level_zero/common.hpp

```diff
@@ -21,6 +21,7 @@
 #include <ze_api.h>
 #include <zes_api.h>
 
+#include <sycl/detail/defines.hpp>
```

Comment:
Simiar comment about using `sycl` namespace, causes issues for out-of-tree adapter builds, see https://github.com/intel/llvm/pull/10135.

Reply:
This has now been removed since it was only required for the `pi::die` impl that I've removed.